### ISSUE TITLE
Add Redstone channel support to logistics pipe interfaces

### ISSUE THREAD

**Issue by Targren** [TeamPneumatic/pnc-repressurized#1016]
### Describe the feature

Just the usual "on with Redstone"/"off with Redstone"/"ignore" trinity (maybe an "activate on pulse" mode, if you're feeling sassy) and color  channel selector, like the one found on the (un-upgraded) Redstone connectors. 

### Reasons why it should be considered

It would allow more fine grained logistics control.

It's probably easiest to go by example: In a 4-chamber refinery processing oil, one could use comparator signals (or other mods, such as Refined Storage's "Detector") to redirect LPG output to storage/fluid void (activating a requester) and disabling delivery of mix-ins (coal in E6E, e.g. Not sure if that's the stock recipe), while letting the refinery continue to run to keep producing lubricant.

**Comment by desht** [comment]
So enabling/disabling logistics frames based on a neighbouring redstone signal?  Not sure how color channels come into this, though?

The scenario you describe (LPG to fluid or void) can already be done, though:
* Active Provider Frame on the refinery output
* Storage Frame on the fluid tank, filtered for LPG
* Default Storage Frame on a fluid void, also filtered for LPG

**Comment by Targren** [comment]
I was thinking that it would let them be integrated into more complex logistical setups (which I did an admittedly poor job of describing at 4 am). I guess if they'll react to a neighboring signal, , the RS color channel support isn't strictly necessary, since you can just point a Redstone Module at it, set to the appropriate channel. 

I guess it's more just a question of how much work it is to implement it into the frame itself, for the sake of "neater" builds, but yeah, "external" redstone control would be just fine, too.

**Comment by desht** [comment]
Bearing in mind that frames don't connect to tubes, I'm not sure how color channel support would work.

But having them react to incoming redstone on the block they're on is a possibility, yes.

**Comment by Targren** [comment]
They don't? Well, we can chalk that up to me misunderstanding how they work, then. :)

**Comment by desht** [comment]
Yeah Logistics Modules (which go on tubes) point at frames, but frames can also be used with Logistics Drones, no tubes involved at all...
